slack-app-backend/src/main/java/com/slack/api/app_backend/events/handler/AppDeletedHandler.java

```diff
@@ -0,0 +1,13 @@
+package com.slack.api.app_backend.events.handler;
```

Comment:
These handlers are legacy ones (meaning these are not used in bolt-java), but I've added them this time. In the future, we may want to deprecate them since v2.